### DISCUSSION TITLE
EssentialX Discord add Player Count feature after Player join or left

### DISCUSSION THREAD

**Discussion by kvintanaka** [EssentialsX/Essentials#4424] [Ideas and Feature Suggestions] [answered]
Good day, EssentialsX Dev
I have a suggestion to add player count after player joined or player have left on the Discord server

Would this be possible?

Regards,
Kevin Tanaka

**Comment by JRoy** [comment, marked as answer]
Will be added in https://github.com/EssentialsX/Essentials/pull/4427
